Azure/azure-cli
# [Webapp] Bugfix: az webapp log deployment show - return deployment logs instead of log metadata

**Description**  
closes #14099 
- az webapp log deployment show should show actual logs instead of the log info
- Mark commands as in preview

**Testing Guide**  
`az webapp log deployment show`

Should have output of a list of logs similar to:

[
{
"log_time": "2020...",
"id": "id",
"message": "log message",
"type": 0,
"details_url": null
},
{
"log_time": "2020...",
"id": "id",
"message": "log message 2",
"type": 0,
"details_url": null
}
]

Rather than an object similar to

{
    "active": false,
    "author": "N/A",
    "author_email": "N/A",
    "complete": true,
    "deployer": "GITHUB_ZIP_DEPLOY",
    "end_time": "2020-06-25T19:04:18.8758562Z",
    "id": "id",
    "is_readonly": true,
    "is_temp": false,
    "last_success_end_time": "2020-06-25T19:04:18.8758562Z",
    "log_url": "https://calvin-githubactions-webapp.scm.azurewebsites.net/api/deployments/deploymentId/log",
    "message": "Created via a push deployment",
    "progress": "",
    "received_time": "2020-06-25T19:04:00.9811072Z",
    "site_name": "calvin-githubactions-webapp",
    "start_time": "2020-06-25T19:04:01.061354Z",
    "status": 4,
    "status_text": "",
    "url": "https://calvin-githubactions-webapp.scm.azurewebsites.net/api/deployments/deploymentId"
  }

---

This checklist is used to make sure that common guidelines for a pull request are followed.
